Sources/Public/iOS/LottieAnimationViewBase.swift

```diff
@@ -39,7 +39,11 @@ open class LottieAnimationViewBase: UIView {
 
   var screenScale: CGFloat {
     #if os(iOS) || os(tvOS)
-    UIScreen.main.scale
+    if #available(iOS 13.0, tvOS 13.0, *) {
+        return window?.windowScene?.screen.scale ?? 1.0
+    } else {
+        return UIScreen.main.scale
+    }
```

Comment:
The linter check is failing because these blocks are indented with 4 spaces instead of 2. You can run the linter yourself with `bundle exec rake format:swift`.

```suggestion
    if #available(iOS 13.0, tvOS 13.0, *) {
      return window?.windowScene?.screen.scale ?? 1.0
    } else {
      return UIScreen.main.scale
    }
```